### DISCUSSION TITLE
Gas-Water Flow Velocity Distribution using OpenPNM and PoreSpy

### DISCUSSION THREAD

**Discussion by gxw-ll** [PMEAL/OpenPNM#2856] [Q&A]
Dear Professor，
I am currently engaged in the analysis of gas-water flow simulation and have obtained a set of relative permeabilities. I am interested in utilizing gas-water flow distribution to interpret the gas-water phase invasions.
Could you please provide insights on whether OpenPNM and PoreSpy are suitable for such an analysis? Any guidance or information you can offer would be greatly appreciated.

**Comment by likeduck** [comment]
According to my limited knowledge, OpenPNM can complete the visualization of invasion by IP algorithm.

**Comment by jgostick** [comment]
Here is a sample of what openpnm can do: https://openpnm.org/examples/tutorials/09_simulating_invasion.html

And here is a sample of using porespy for image based work: https://porespy.org/examples/simulations/tutorials/drainage_with_gravity_advanced.html
